db/schema.rb

```diff
@@ -48,6 +48,8 @@
     t.datetime "updated_at", null: false
     t.string "verification_status"
     t.string "verification_details"
+    t.boolean "shown_verification_failed_modal", default: false
```

Comment:
Are these legacy? Can you recreating the db so we have the most up to date schema?

Reply:
I must have added wrong hunk when rebasing. I'll fix.